docs/vm/create-vm.md

```diff
@@ -11,32 +11,105 @@ keywords:
 
 1. Choose the option to create either one or multiple VM instances.
+1. Select the namespace of your VMs, only the `harvester-public` namespace is visible to all users.
 1. The VM Name is a required field.
-1. The VM Template is optional. You can select ISO, raw, and Windows image templates as default options.
-1. Configure the CPU and Memory of the VM.
+1. (Optional) VM template is optional, you can choose `iso-image`, `raw-image` or `windows-iso-image` template to speed up your VM instance creation.
+1. Configure the virtual machine's CPU and memory (see [overcommit settings](/settings/settings/#overcommit-config) if you want to over-provision).
 1. Select SSH keys or upload new keys.
 1. Select a custom VM image on the **Volumes** tab. The default disk will be the root disk. You can add more disks to the VM.
-1. To configure networks, go to the **Networks** tab. The **Management Network** is added by default. It is also possible to add secondary networks to the VMs using VLAN networks. You may configure these on **Advanced > Networks**.
-1. Advanced options such as hostname and cloud-init data are optional. You may configure these in the **Advanced Options** section.
+1. To configure networks, go to the **Networks** tab. 
+    1. The **Management Network** is added by default, you can remove it if the VLAN network is configured.
+    1. You can also add additional networks to the VMs using VLAN networks. You may configure the VLAN networks on **Advanced > Networks** first.
+1. Advanced options such as run strategy, os type and cloud-init data are optional. You may configure these in the **Advanced Options** section when applicable.
 
 ![create-vm](assets/create-vm.png)
 
-![create-vm](assets/choose-vm-image.png)
+## Volumes
+
+You can add one or more additional volumes via the `Volumes` tab, by default the first disk will be the root disk, you can change the boot order by dragging and dropping volumes, or using the arrow buttons.
+
+A disk can be made accessible via the following types:
+
+| type   | description                                                                                    |
+|:--------|:-----------------------------------------------------------------------------------------------|
+| disk   | A disk disk will expose the volume as an ordinary disk to the VM.                              |
+| cd-rom | A cd-rom disk will expose the volume as a cd-rom drive to the VM. It is read-only by default.  |
+
+![create-vm](assets/create-vm-volumes.png)
+
+!!! note "Container Disk"
+    `Container disks` are ephemeral storage devices that can be assigned to any number of VMs. This makes them an ideal tool for users who want to replicate a large number of VM workloads or inject machine drivers that do not require persistent data.
+
+    Note: Container disks are not a good solution for any workload that requires persistent root disks across VM restarts.
```

Comment:
Just curious, does the ephemeral storage also vanish after live migration? Or it's a forbidden operation.

Reply:
I think so, live migration will only migrate those disks that support RWX mode.